Formula/pygtkglext.rb

```diff
@@ -28,7 +28,8 @@ def install
   end
 
   test do
-    ENV.append_path "PYTHONPATH", Formula["pygtk"].opt_lib+"python2.7/site-packages/gtk-2.0"
-    system "python", "-c", "import pygtk", "pygtk.require('2.0')", "import gtk.gtkgl"
+    ENV.append_path "PYTHONPATH", Formula["pygtk"].opt_lib/"python2.7/site-packages/gtk-2.0"
+    system Formula["python@2"].opt_bin/"python2.7", "-c", "import pygtk",
```

Comment:
why not use system Python?

Reply:
Our `python@2` is part of the recursive deps tree via `pygobject` and `pygobject` uses `system Formula["python@2"].opt_bin/"python2.7"`, so essentially I went for consistency.

Reply:
That is probably a bogus use of Homebrew Python so I don't think we need to further lock it in.

Reply:
I'd probably just make it `python2.7`